### PR TITLE
Use Microsoft Package to handle toast notification for registry 

### LINKED ISSUE #964: Notification not registered in the system
### Discussed in https://github.com/Flow-Launcher/Flow.Launcher/discussions/946

<div type='discussions-op-text'>

<sup>Originally posted by **stax76** January  1, 2022</sup>
I disabled Flow Launcher notifications system-wide in Windows 11, I think I did this with a UI setting, and now I'm not able to re-enable notifications, I'm not getting a notification when I press F5 using v1.9.1. I've searched the Windows 11 settings UI and the registry but could not find a way to be re-enabled notifications. In the Windows settings under 'System > Notifications' Flow Launcher is not listed, I tried reinstalling, but it did not help.</div>

### PR DESCRIPTION
**Pull request by taooceros**
#946
close #964

### CODE REVIEW

File: Flow.Launcher/Notification.cs [hunk 1]
```diff
@@ -8,10 +9,17 @@ namespace Flow.Launcher
 {
     internal static class Notification
     {
+        internal static bool legacy = Environment.OSVersion.Version.Build < 19041;
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Interoperability", "CA1416:Validate platform compatibility", Justification = "<Pending>")]
+        internal static void Uninstall()
+        {
+            if (!legacy)
+                ToastNotificationManagerCompat.Uninstall();
```
Comment (jjw24) [inline Flow.Launcher/Notification.cs:17]: What does this Uninstall do?

File: Plugins/Flow.Launcher.Plugin.Shell/Main.cs [hunk 2]
```diff
@@ -317,7 +317,7 @@ public void Init(PluginInitContext context)
 
         bool API_GlobalKeyboardEvent(int keyevent, int vkcode, SpecialKeyState state)
         {
-            if (_settings.ReplaceWinR)
+            if (!context.CurrentPluginMetadata.Disabled && _settings.ReplaceWinR)
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Shell/Main.cs:320]: Is it easy to full disable the plugin without doing it here? Obviously taking into consideration that disabled plugin still needs to be shown in the Settings window

File: Flow.Launcher/Notification.cs (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher/Notification.cs:17]: https://docs.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/send-local-toast?tabs=uwp#step-4-handling-uninstallation

File: Plugins/Flow.Launcher.Plugin.Shell/Main.cs (hunk 2 above)
Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.Shell/Main.cs:320]: Not easy due to the current model. We cannot make sure the plugin won't do anything or register anything under its constructor or Init method, so we can only assume that.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**